sync/src/main/java/tech/pegasys/teku/sync/AttestationManager.java

```diff
@@ -40,42 +41,42 @@
 
   private final PendingPool<DelayableAttestation> pendingAttestations;
   private final FutureItems<DelayableAttestation> futureAttestations;
+  private final Consumer<Attestation> attestationPoolConsumer;
 
   AttestationManager(
       final EventBus eventBus,
       final ForkChoiceAttestationProcessor attestationProcessor,
       final PendingPool<DelayableAttestation> pendingAttestations,
-      final FutureItems<DelayableAttestation> futureAttestations) {
+      final FutureItems<DelayableAttestation> futureAttestations,
+      final Consumer<Attestation> attestationPoolConsumer) {
```

Comment:
Is there a reason this doesn't just take the `AggregatingAttestationPool`? Then `AttestationManager` would really be heading towards the thing that encompasses all attestation components like I'd hoped for.